registry.k8s.io/images/charts/images.yaml

```diff
@@ -1,3 +1,6 @@
+- name: dra-example-driver
+  dmap:
+    "sha256:ab5cd9fd898b2ff79bea030520170a7feb579ff463dce9758775f7cce7d7231f": ["0.1.3"]
```

Comment:
```
helm template --disable-openapi-validation oci://us-central1-docker.pkg.dev/k8s-staging-images/charts/dra-example-driver --version 0.1.3 --kube-version 1.32 --set allowDefaultNamespace=true| grep image:
Pulled: us-central1-docker.pkg.dev/k8s-staging-images/charts/dra-example-driver:0.1.3
Digest: sha256:ab5cd9fd898b2ff79bea030520170a7feb579ff463dce9758775f7cce7d7231f
        image: registry.k8s.io/dra-example-driver/dra-example-driver:v0.1.0
```

I verified this image. Btw, why do you not add the `v` prefix to the chart version?

Reply:
Helm expects versions not to have the `v` prefix. I did some investigation about that here: https://github.com/kubernetes-sigs/kueue/issues/4445

Reply:
Oh, I see. So, this chart version without the `v` prefix allows us to download the latest chart by download command without a specific version flag, right?

Reply:
Yes, and also all the other syntax the `--version` flag accepts to match specific ranges of versions: https://pkg.go.dev/github.com/Masterminds/semver/v3#hdr-Checking_Version_Constraints_and_Comparing_Versions

Reply:
Interesting. Thank you for clarifying.
/approve

/hold in case having comment from @upodroid